pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -374,3 +419,23 @@ func engineMonitorService(engineMonitor *v1.Service, r *ReconcileChaosEngine, re
 	reqLogger.Info("Skip reconcile: engineMonitor Service already exists", "Service.Namespace", service.Namespace, "Service.Name", service.Name)
 	return nil /*You can return now, both sec resources are existing */
 }
+
+// engineExporterPod to Check if the engineExporter Pod is already exists, else create
+func engineExporterPod(engineExporter *v1.Pod, r *ReconcileChaosEngine, reqLogger logr.Logger) error {
+	pod := &corev1.Pod{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: engineExporter.Name, Namespace: engineExporter.Namespace}, pod)
+	if err != nil && k8serrors.IsNotFound(err) {
+		reqLogger.Info("Creating a new engineExporter Pod", "Pod.Namespace", engineExporter.Namespace, "Pod.Name", engineExporter.Name)
+		err = r.client.Create(context.TODO(), engineExporter)
+		if err != nil {
+			return err
+		}
+
+		// Pod created successfully - don't requeue
```

Comment:
Delete this comment, Not required

Reply:
Every reconcile function, has it, and I think it's needed